test/integration/projects/controls/gcloud_projects.rb

```diff
@@ -191,7 +191,7 @@
             expect(data['bindings'][0]['members']).to include(
               "serviceAccount:#{cloudbuild_sa[business_unit]}"
             )
-            expect(data['bindings'][0]['role']).to eq "roles/iam.serviceAccountTokenCreator"
+            expect(data['bindings'][0]['role']).to eq ["roles/iam.serviceAccountTokenCreator", "roles/editor"]
```

Comment:
We should actually check for "roles/editor" on the SA project factory creates `#{base_project_sa[environment_code][business_unit]}`

Reply:
Got it, thanks @bharathkkb, I'll update this.